### PR TITLE
Adding Symfony 6 support

### PR DESCRIPTION
**Pull request by Romaixn**
Adding Symfony 6 support in composer.json file (view #1839)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The following table shows the compatibilities of different versions of the bundl
 
 | FOSElasticaBundle                                                                       | Elastica | Elasticsearch | Symfony    | PHP   |
 | --------------------------------------------------------------------------------------- | ---------| ------------- | ---------- | ----- |
-| [6.0] (master)                                                                          | ^7.1     | 7.\*          | ^4.4\|^5.3 | >=7.4 |
+| [6.0] (master)                                                                          | ^7.1     | 7.\*          | ^4.4\|^5.3|^6.0 | >=7.4 |
```
Comment (deguif) [inline README.md:38]: You should escape the `|` character inside a table cell

Comment (Romaixn) [reply README.md]: Oh yeah missed it, thanks fixed it !

**Comment by olivier1980** [comment]
What is the holdup here..the coding style error is only showing some code that moved but is pretty much identical otherwise?

**Comment by franmomu** [comment]
IMO we should also add a CI job testing with Symfony 6

**Comment by XWB** [comment]
@olivier1980 I cannot merge until all tests are green.

**Comment by olivier1980** [comment]
> @olivier1980 I cannot merge until all tests are green.

Yeah I know, what Im curious about is why it fails and if it can be fixed. Error seens to be 'composer.json is not normalized'?

**Comment by natewiebe13** [comment]
@olivier1980 it's because composer.json is not in the expected order (the workflow log shows the changes needed). Rebasing the PR would resolve this. That being said, it looks like #1844 deals with this and adds more tests.

**Comment by mamchyts** [comment]
ping

cc @Romaixn

**Comment by Romaixn** [comment]
> ping
> 
> cc @Romaixn

I think #1844 answers this RP in a much better way! I close this PR for the moment 🥇

**Pull request closed without merging**